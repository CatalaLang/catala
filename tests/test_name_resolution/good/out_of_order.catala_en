## Article

```catala
declaration scope S:
  context a content A
  context b content B

declaration structure A:
   data x content integer
   data y content B

declaration structure B:
   data y content boolean
   data z content decimal

scope S:
   definition b equals B { -- y: true -- z: 0.0}
   definition a equals A { -- x: 0 -- y : b }
```
