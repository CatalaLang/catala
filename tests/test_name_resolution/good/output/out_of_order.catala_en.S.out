[RESULT] Computation successful! Results:
[RESULT] a = A {"x": 0, "y": B {"y": true, "z": 0.}}
[RESULT] b = B {"y": true, "z": 0.}
